### PR TITLE
Settings for being able to upload

### PR DESCRIPTION
**Pull request by trustlyandersisraelsson**
I have tried to take inspiration from those guides:
https://docs.github.com/en/actions/publishing-packages/publishing-docker-images
https://theoverengineered.blog/posts/publishing-my-first-artifact-to-maven-central-using-github-actions

### CODE REVIEW

**Review by BergJonatan — COMMENTED** [review]
Do we need SCM information in the pom?
https://central.sonatype.org/publish/requirements/#scm-information

**Comment by trustlyandersisraelsson** [comment]
> Do we need SCM information in the pom? https://central.sonatype.org/publish/requirements/#scm-information

Hmm, many of the guides I've looked at doesn't include that. I'm guessing that the `nexus-staging-maven-plugin` do it for us.

**Review by niclasivarsson — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -69,12 +80,63 @@
...
+        <plugins>
+            <plugin>
+                <groupId>org.sonatype.plugins</groupId>
+                <artifactId>nexus-staging-maven-plugin</artifactId>
+                <version>1.6.8</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <serverId>ossrh</serverId>
+                    <nexusUrl>https://oss.sonatype.org/</nexusUrl>
+                    <autoReleaseAfterClose>true</autoReleaseAfterClose>
```
Comment (niclasivarsson) [inline pom.xml:93]: Might want to hold off on this and manually inspect the release to make sure it looks the way we want first.

Comment (trustlyandersisraelsson) [reply pom.xml]: Thats true. Default seams to be false, I'm removing it

**Pull request merged**